pom.xml

```diff
@@ -510,6 +513,18 @@
                           <ignoreClass>javax.transaction.*</ignoreClass>
                         </ignoreClasses>
                       </dependency>
+                      <dependency>
+                        <!-- The openshift-client duplicates these classes from the kubernetes-client.
```

Comment:
@errantepiphany do you mind open an issue with io.fabric8 and link it to the GUVNOR-3023 jira? so we can track it